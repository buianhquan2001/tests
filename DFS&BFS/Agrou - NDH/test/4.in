100
12 5
8 6 wolf
2 1 villager
11 7 wolf
2 3 villager
11 8 villager
16 1
5 8 villager
10 6
10 2 villager
10 4 wolf
3 6 villager
9 1 villager
8 1 villager
9 4 villager
19 7
19 4 wolf
2 13 wolf
2 4 wolf
10 19 wolf
9 15 villager
17 13 villager
8 13 wolf
19 3
13 5 villager
7 15 wolf
3 19 wolf
17 3
13 16 villager
3 12 villager
14 15 wolf
19 3
9 3 wolf
8 7 villager
13 14 wolf
19 3
17 3 villager
19 13 villager
5 4 villager
10 6
10 5 villager
10 8 villager
9 8 wolf
9 8 wolf
3 6 villager
4 1 villager
15 9
2 10 villager
7 3 wolf
4 9 wolf
9 1 wolf
6 11 wolf
1 3 wolf
1 3 villager
7 3 villager
9 8 wolf
14 3
6 2 villager
6 4 villager
14 9 villager
10 6
5 6 villager
7 6 villager
8 7 villager
10 7 villager
5 6 villager
8 9 villager
20 6
5 14 villager
8 16 wolf
5 8 wolf
11 14 wolf
15 6 wolf
3 17 villager
16 3
3 12 wolf
3 6 wolf
8 7 villager
10 6
7 1 wolf
7 4 villager
7 4 wolf
2 9 wolf
5 10 wolf
7 5 villager
19 8
19 9 wolf
19 3 villager
4 2 villager
14 17 wolf
12 3 villager
12 13 wolf
4 15 wolf
3 14 wolf
13 5
7 3 villager
5 3 villager
7 4 villager
5 2 wolf
10 7 wolf
18 8
1 9 wolf
8 10 wolf
5 4 villager
6 12 wolf
18 11 wolf
14 18 villager
13 9 villager
6 10 wolf
16 6
12 11 villager
13 11 villager
11 14 villager
11 2 wolf
7 14 wolf
12 14 villager
19 10
11 17 wolf
19 10 wolf
16 3 villager
9 13 villager
2 1 villager
7 6 wolf
19 13 villager
5 14 wolf
5 16 wolf
19 16 villager
15 1
15 11 wolf
16 5
15 5 villager
2 5 wolf
12 1 wolf
10 16 villager
15 10 villager
20 5
2 12 villager
2 8 villager
15 3 wolf
13 17 villager
18 1 wolf
12 10
10 6 wolf
4 5 wolf
8 9 villager
6 4 villager
12 7 wolf
4 7 wolf
1 5 villager
10 6 wolf
6 1 wolf
6 7 wolf
15 4
10 5 wolf
13 3 wolf
5 11 wolf
15 4 wolf
19 10
8 1 villager
8 3 wolf
9 8 wolf
15 5 wolf
11 14 wolf
4 13 wolf
9 14 wolf
14 2 villager
19 16 villager
14 3 villager
20 3
2 1 wolf
2 1 villager
11 10 wolf
18 8
14 16 villager
13 18 wolf
15 16 villager
15 6 wolf
10 17 wolf
10 12 wolf
5 8 wolf
15 9 wolf
20 8
5 4 villager
20 8 wolf
20 14 villager
5 2 villager
18 7 villager
12 16 villager
5 11 wolf
12 13 wolf
11 3
6 9 villager
6 1 villager
11 10 villager
11 9
11 5 villager
1 10 wolf
11 1 villager
8 6 villager
11 1 villager
1 6 wolf
11 5 villager
11 6 villager
4 5 villager
18 4
15 3 villager
8 4 villager
1 9 wolf
14 13 villager
16 7
3 6 wolf
7 10 villager
7 6 villager
1 10 villager
14 16 wolf
14 16 wolf
1 16 wolf
12 7
8 6 wolf
11 1 wolf
3 9 villager
7 12 wolf
6 1 villager
6 12 wolf
7 5 wolf
10 4
4 8 villager
3 1 villager
6 9 wolf
9 7 wolf
13 4
13 12 wolf
5 11 villager
13 1 wolf
8 4 villager
14 1
13 8 wolf
16 5
8 5 wolf
8 4 villager
14 4 wolf
16 10 wolf
16 8 villager
13 6
6 9 wolf
1 3 villager
11 10 villager
7 9 villager
7 4 wolf
6 12 villager
18 4
10 18 wolf
4 17 villager
8 3 villager
1 3 villager
10 4
8 9 villager
7 3 wolf
4 7 villager
10 1 wolf
18 3
15 9 villager
14 6 villager
15 18 wolf
17 8
13 15 wolf
14 8 wolf
1 9 villager
12 15 villager
12 15 villager
14 5 wolf
17 10 villager
13 15 wolf
16 5
1 14 wolf
14 4 wolf
8 7 wolf
13 1 wolf
14 6 villager
19 4
13 19 villager
7 1 wolf
15 16 wolf
16 10 villager
15 2
2 3 wolf
8 3 wolf
12 1
3 10 wolf
15 1
15 14 wolf
10 7
9 7 wolf
8 6 wolf
10 6 villager
1 3 villager
9 2 wolf
7 4 wolf
10 8 wolf
16 2
3 10 villager
3 9 villager
20 5
18 9 villager
19 20 wolf
19 17 wolf
6 19 villager
18 13 wolf
15 2
11 1 wolf
14 9 wolf
14 10
12 9 villager
12 5 wolf
12 5 villager
12 14 villager
4 12 wolf
4 8 villager
2 1 villager
4 8 wolf
11 8 wolf
12 3 wolf
12 7
5 4 wolf
6 10 villager
2 7 villager
6 4 wolf
1 9 wolf
11 4 villager
11 5 villager
18 6
4 2 wolf
13 18 villager
4 1 wolf
4 11 wolf
10 12 villager
10 6 villager
18 9
3 18 villager
16 12 villager
14 5 wolf
11 15 wolf
18 16 wolf
9 8 villager
14 17 wolf
14 7 wolf
12 13 villager
18 10
11 3 wolf
18 14 wolf
5 3 villager
6 12 wolf
6 18 villager
11 3 wolf
9 13 villager
9 7 wolf
15 18 wolf
4 10 wolf
12 9
12 5 villager
10 2 wolf
8 5 villager
11 3 villager
7 3 villager
11 4 villager
2 3 wolf
10 8 wolf
8 5 wolf
20 9
3 14 wolf
4 20 wolf
3 11 wolf
18 2 wolf
13 14 wolf
3 14 villager
18 15 wolf
7 14 wolf
1 11 wolf
11 5
5 10 wolf
5 8 villager
11 3 villager
7 1 wolf
4 7 villager
15 10
6 9 wolf
6 5 villager
14 4 wolf
11 9 villager
8 10 wolf
8 3 wolf
10 5 villager
7 14 villager
7 11 villager
14 12 villager
14 4
14 11 wolf
9 12 villager
13 8 villager
8 4 villager
11 2
10 11 villager
10 9 wolf
11 5
2 4 wolf
11 10 wolf
5 6 villager
7 8 villager
3 11 wolf
16 5
3 12 villager
4 5 villager
8 1 villager
12 16 wolf
3 9 villager
13 8
12 1 wolf
12 1 villager
3 1 wolf
11 13 wolf
4 7 wolf
5 6 wolf
2 13 wolf
5 9 villager
13 7
2 5 wolf
4 6 villager
4 6 villager
11 6 villager
8 9 wolf
11 7 villager
8 13 wolf
10 8
4 7 wolf
9 3 wolf
8 2 wolf
2 6 wolf
9 6 wolf
1 6 wolf
2 6 wolf
2 3 wolf
16 5
5 7 villager
3 15 villager
16 13 wolf
5 8 wolf
10 15 wolf
20 2
6 1 wolf
2 19 wolf
13 3
7 2 wolf
6 4 wolf
7 5 villager
18 5
14 5 wolf
4 15 villager
2 6 wolf
3 16 wolf
10 6 wolf
20 2
15 20 villager
15 16 villager
15 10
14 7 villager
5 11 wolf
5 2 villager
4 11 wolf
1 14 wolf
1 11 villager
5 8 villager
10 15 wolf
14 11 wolf
4 9 villager
10 2
10 9 wolf
2 7 wolf
18 9
6 1 villager
4 18 wolf
7 18 wolf
1 10 villager
18 2 wolf
5 17 wolf
16 9 villager
4 13 villager
4 12 wolf
15 2
3 15 wolf
12 2 wolf
17 6
7 15 wolf
3 9 wolf
16 6 wolf
5 7 wolf
2 14 wolf
2 14 villager
15 2
2 13 villager
2 10 wolf
16 9
13 6 wolf
16 4 villager
13 10 villager
13 3 villager
14 13 wolf
5 2 wolf
5 13 villager
16 15 villager
1 4 wolf
14 5
6 4 wolf
5 14 villager
10 3 wolf
8 1 villager
5 4 wolf
18 4
12 6 wolf
9 16 villager
7 15 villager
7 4 wolf
15 10
10 4 villager
7 11 wolf
1 2 wolf
7 9 villager
9 4 villager
15 6 wolf
1 6 wolf
3 10 villager
14 6 wolf
14 2 wolf
13 8
2 11 wolf
2 9 villager
8 1 wolf
2 7 villager
2 4 villager
2 8 wolf
2 9 wolf
11 5 wolf
16 6
16 6 villager
3 2 villager
1 14 wolf
12 4 wolf
3 14 villager
16 8 villager
19 7
2 17 wolf
12 10 villager
7 18 villager
3 8 wolf
15 17 villager
7 18 villager
9 12 wolf
20 10
10 19 villager
10 13 wolf
10 17 wolf
10 17 villager
10 15 villager
16 10 villager
18 8 wolf
17 8 villager
6 9 villager
17 8 wolf
17 7
14 13 wolf
4 8 villager
9 11 wolf
17 14 wolf
10 16 villager
7 12 wolf
9 14 wolf
14 1
2 11 wolf
18 2
2 9 wolf
2 8 wolf
18 2
2 8 villager
8 6 villager
15 6
5 6 wolf
14 5 wolf
15 12 wolf
14 6 villager
14 10 villager
7 13 villager
13 8
3 4 villager
12 11 wolf
5 13 wolf
12 4 wolf
10 1 wolf
3 6 wolf
3 11 wolf
8 4 wolf
19 2
8 19 wolf
15 10 villager
10 5
2 1 villager
7 4 villager
2 8 wolf
7 6 villager
3 8 villager
19 3
6 11 wolf
16 13 wolf
10 2 villager
19 10
8 9 villager
12 17 villager
14 9 wolf
18 9 wolf
13 17 villager
14 8 wolf
8 6 villager
13 19 villager
8 5 wolf
13 15 villager
15 3
3 14 wolf
2 6 wolf
3 10 wolf
16 3
4 7 wolf
10 7 villager
9 7 wolf
14 10
9 7 wolf
5 10 villager
5 4 villager
5 11 villager
10 14 villager
7 12 wolf
9 7 villager
14 7 wolf
9 1 wolf
10 8 villager
