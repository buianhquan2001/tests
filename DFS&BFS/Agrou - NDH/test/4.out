10
16
8
16
17
16
17
19
9
-1
14
10
17
15
-1
15
11
14
14
15
14
14
18
9
12
-1
-1
13
17
11
-1
17
14
9
9
12
13
13
11
17
8
17
15
13
17
14
11
14
7
16
17
13
-1
-1
17
14
13
-1
-1
9
10
13
10
9
15
-1
11
6
13
18
11
15
20
12
8
15
13
-1
14
12
12
16
12
-1
14
16
-1
14
13
17
18
13
-1
18
8
17
16
13
14
-1
